src/user-management/user-management.ts

```diff
@@ -48,9 +48,13 @@ import {
   Challenge,
   ChallengeResponse,
   Factor,
```

Comment:
does this still need to use the MFA standalone Factor?

Reply:
ah yeah had some lingering usage, fixed now